### DISCUSSION TITLE
Prune / mass delete endpoint

### DISCUSSION THREAD

**Discussion by Darkflame72** [stoatchat/discussions#179] [Feature Suggestions]
Add a prune or mass delete endpoint which would allow the mass deletion of x number of messages in a channel. This will be especially useful to moderation bots to cleanup after spam or other situations. It will also stop there being a lot of requests to mass delete messages and instead can be a single one.

**Comment by DeclanChidlow** [comment]
https://developers.revolt.chat/developers/api/reference.html#tag/messaging/DELETE/channels/{target}/messages/bulk
